frontend/src/metabase/public/containers/PublicDashboard.jsx

```diff
@@ -151,15 +151,7 @@ export default class PublicDashboard extends Component {
         parameterValues={parameterValues}
         setParameterValue={this.props.setParameterValue}
         actionButtons={
-          buttons.length > 0 && (
-            <div>
-              {buttons.map((button, index) => (
-                <span key={index} className="m1">
-                  {button}
-                </span>
-              ))}
-            </div>
-          )
+          buttons.length > 0 && <div className="flex">{buttons}</div>
```

Comment:
![](https://media1.giphy.com/media/QhGh5jZXzMEkmw8ekj/giphy.gif)

Reply:
Just so I understand, why did we need to map over buttons before and don't have to do it anymore?
Flex will line them up nicely, but we have to rely on the spacing/styles defined in each of the buttons, right?

Reply:
Yes, you're right. These buttons got their own spacing during the development of the adding questions sidebar